portworx/metadata.csv

```diff
@@ -35,6 +54,12 @@ portworx.pool_stats.pool_written_bytes,gauge,,byte,,Pool stats for written bytes
 portworx.proc_stats.cpu_percenttime,gauge,,,,Proc stats for CPU percent time,0,portworx,cpu percent time,
 portworx.proc_stats.res,gauge,,,,Proc stats for resident memory size,0,portworx,resident memory,
 portworx.proc_stats.virt,gauge,,,,Proc stats for virtual memory size,0,portworx,virtual memory,
+portworx.pool_stats.status,gauge,,,,"Status of this Pool (0=Offline,1=Online,2=Full,3=NotFound,4=Maintenance)",0,portworx,pool status,
```

Comment:
```suggestion
portworx.pool_stats.status,gauge,,,,"Status of this pool (0=Offline, 1=Online, 2=Full, 3=NotFound, 4=Maintenance)",0,portworx,pool status,
```

Reply:
I'm not sure if NotFound is how the status is returned, so leaving as-is (instead of suggesting that we change it to "Not Found").